app/update_version.go

```diff
@@ -7,9 +7,11 @@ import (
 
 func UpdateVersion(nsConfig api.Config, appId int64, envName, version string) error {
 	client := api.Client{Config: nsConfig}
-	_, err := client.AppEnvs().Update(appId, envName, version)
+	result, err := client.AppEnvs().Update(appId, envName, version)
 	if err != nil {
 		return fmt.Errorf("error updating app version: %w", err)
+	} else if result == nil {
+		return fmt.Errorf("could not find application environment")
```

Comment:
Did you have a scenario where a nil result came back? I would think that this isn't possible when looking at the endpoint code. If it were, I'm not sure that is the right error message. I would think the error message would say something about this being unexpected.

Reply:
Same as below.
Any thoughts on how to improve the messaging?